Frontend/ui-driver/src/Screens/ReferralScreen/View.purs

```diff
@@ -347,8 +349,19 @@ dateAndTime push state =
         , gravity RIGHT
         , weight 1.0
         ]
-    ]
+  ]
 
+leaderBoardRanksCover :: forall w . (Action -> Effect Unit) -> ST.ReferralScreenState -> PrestoDOM (Effect Unit) w
```

Comment:
@Anuragini-2506 Can we add `swipeRefreshLayout` to `leaderBoardRanks` function itself instead of creating another function `leaderBoardRanksCover`

Reply:
If we add the swipeRefreshLayout inside the leaderBoardRanks, the views are not visible. That's why I created a wrapper around it. Normally the swipeRefreshLayout is taking the prestoDOM.List, but this view is not having prestoDOM.list that's why I created a new wrapper around it